packages/ui/src/components/va-accordion/VaAccordion.stories.ts

```diff
@@ -52,6 +52,38 @@ Default.play = async ({ canvasElement, step }) => {
   })
 }
 
+export const vModel = () => ({
+  components: { VaAccordion, VaCollapse },
+  data: () => ({ value: [true, false, true] }),
+  template: `
+    <va-accordion v-model="value" multiple>
+      <va-collapse
+        v-for="i in 3"
+        :key="i"
+        :header="value[i - 1] ? 'Must be opened' : 'Must be closed'"
```

Comment:
Let's not have any text description in story apart from what is absolutely necessary
```suggestion
        header="Collapse"
```

Reply:
Previously it was hidden even if header was "Must be opened". So, at least it helped me to fix a bug. More, it probably will prevent this bug in future.